test/multiverse/suites/async_http/Envfile

```diff
@@ -16,4 +16,4 @@ def gem_list(async_http_version = nil)
   GEM_LIST
 end
 
-create_gemfiles(ASYNC_HTTP_VERSIONS)
+create_gemfiles(ASYNC_HTTP_VERSIONS) unless defined?(JRUBY_VERSION) # async-http needs native extensions
```

Comment:
This works as-is, but Envfiles generally skip JRuby using the `Multiverse::Envfile#suite_condition` method. Here's how it's used in Sidekiq:
https://github.com/newrelic/newrelic-ruby-agent/blob/c959f40959bffcfc8aca5478926f89ebedbe3308/test/multiverse/suites/sidekiq/Envfile#L5-L7